### PR TITLE
clarity updates

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?**  Clarity updates for create an auth server
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-666914](https://oktainc.atlassian.net/browse/OKTA-666914)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/8113c17d11c0bf6393862f0372405f5a7d5af352/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2753c740-8570-4b76-b87e-14df9b0b9dd3) | 82  | 78  | 95  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.faf81fae-2374-4db3-865e-36edde7dc2be?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ Okta allows you to create multiple custom authorization servers that you can use
...
 
-For simple use cases the default custom authorization server should suffice. If your application has requirements such as requiring more scopes, customizing rules for when to grant scopes, or you need more authorization servers with different scopes and claims, then this guide is for you.
+For simple use cases the default custom authorization server should suffice. If your app has requirements such as requiring more scopes or customizing rules for when to grant scopes, then you another authorization server.
 
-See [Which authorization server should you use](/docs/concepts/auth-servers/#which-authorization-server-should-you-use) for more information on the types of authorization servers available to you and what you can use them for.
+See the [Authorization servers](/docs/concepts/auth-servers/#which-authorization-server-should-you-use) concept for more information on the types of authorization servers available to you and what you can use them for.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:31]:
```suggestion
See the [Authorization servers](/docs/concepts/auth-servers/#which-authorization-server-should-you-use) concept for more information on the types of authorization servers that are available to you and what you can use them for.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 2]
```diff
@@ -38,19 +38,19 @@ See [Which authorization server should you use](/docs/concepts/auth-servers/#whi
...
 
 When you finish, the authorization server's **Settings** tab displays the information that you provided. If you need to [edit any of the information](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServer/), such as [Signing Key Rotation](/docs/concepts/key-rotation/), click **Edit**.
 
 ## Create access policies
 
-Access policies are containers for rules. Each [access policy](/docs/guides/configure-access-policy/) applies to a particular OpenID Connect application. The rules that the policy contains define different access and refresh token lifetimes depending on the nature of the token request.
+Access policies are containers for rules. Each [access policy](/docs/guides/configure-access-policy/) applies to a particular OpenID Connect app. The rules that the policy contains define different access and refresh token lifetimes depending on the nature of the token request.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:47]:
```suggestion
Access policies are containers for rules. Each [access policy](/docs/guides/configure-access-policy/) applies to a particular OpenID Connect (OIDC) app. The rules that the policy contains define different access and refresh token lifetimes depending on the nature of the token request.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 3]
```diff
@@ -38,19 +38,19 @@ See [Which authorization server should you use](/docs/concepts/auth-servers/#whi
...
 
 1. In the Admin Console, go to **Security** > **API**.
 1. On the **Authorization Servers** tab, select the name of an authorization server.
 1. Select **Access Policies**, and then **Add Policy**.
 1. Enter a **Name** and a **Description** for the policy.
-1. Assign the policy to **All clients** or select **The following clients:** and enter the name of the Okta OpenID Connect applications that are covered by this access policy. This field auto-completes the names of your OpenID Connect applications as you type.
+1. Assign the policy to **All clients** or select **The following clients:** and enter the name of the Okta OpenID Connect apps that are covered by this access policy. This field auto-completes the names of your OpenID Connect apps as you type.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:53]:
```suggestion
1. Assign the policy to **All clients** or select **The following clients:** and enter the name of the OIDC apps that are covered by this access policy. This field auto-completes the names of your OIDC apps as you type.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 4]
```diff
@@ -153,7 +153,7 @@ Create ID token claims for OpenID Connect or access tokens for OAuth 2.0:
 
 ## Test the authorization server
 
-After you set up and customize your authorization server, test it by sending any one of the API calls that returns OAuth 2.0 and OpenID Connect tokens.
+After you set up and customize your authorization server, test it. Make a request using any one of the API calls that returns OAuth 2.0 and OpenID Connect tokens.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:156]:
```suggestion
After you set up and customize your authorization server, test it. Make a request using any one of the API calls that returns OAuth 2.0 and OIDC tokens.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 5]
```diff
@@ -165,7 +165,7 @@ The following are a few things that you can try to ensure that your authorizatio
 
 ### OpenID Connect configuration
 
-You can send an API request to the server's OpenID Connect Metadata URI to verify that your server was created and has the expected configuration values: `https://{yourOktaDomain}/oauth2/{authorizationServerId}/.well-known/openid-configuration` using an HTTP client or by typing the URI inside a browser. This returns information about the OpenID configuration of your authorization server.
+You can send an API request to the server's OpenID Connect Metadata URI. This request verifies that your server was created and has the expected configuration values: `https://{yourOktaDomain}/oauth2/{authorizationServerId}/.well-known/openid-configuration` using an HTTP client or by typing the URI inside a browser. This returns information about the OpenID configuration of your authorization server.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:168]:
```suggestion
You can send an API request to the server's OIDC Metadata URI. This request verifies that your server was created and has the expected configuration values: `https://{yourOktaDomain}/oauth2/{authorizationServerId}/.well-known/openid-configuration` using an HTTP client or by typing the URI inside a browser. This returns information about the OIDC configuration of your authorization server.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 6]
```diff
@@ -185,19 +185,19 @@ For more information on this endpoint, see [Get all claims](https://developer.ok
...
-To test your authorization server more thoroughly, you can try a full authentication flow that returns an ID token. To do this, you need a client application in Okta with at least one user assigned to it.
+To test your authorization server more thoroughly, you can try a full authentication flow that returns an ID token. To do this, you need a client app in Okta with at least one user assigned to it.
 
 For more information you can read about:
 
 - [Create OIDC app integrations using AIW](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc)
 - [Assign applications to users](https://help.okta.com/okta_help.htm?id=ext-assign-apps)
 
-You need the following values from your Okta OpenID Connect application, both of which can be found on your application's **General** tab:
+You need the following values from your Okta OpenID Connect app, both of which can be found on your app's **General** tab:
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:195]:
```suggestion
You need the following values from your OIDC app, both of which can be found on your app's **General** tab:
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 7]
```diff
@@ -185,19 +185,19 @@ For more information on this endpoint, see [Get all claims](https://developer.ok
...
 
 - Client ID
 - A valid Redirect URI
 
-After you have an OpenID Connect application set up, and a user assigned to it, you can try the authentication flow.
+After you have an OpenID Connect app set up, and a user assigned to it, you can try the authentication flow.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:200]:
```suggestion
After you have an OIDC app set up, and a user assigned to it, you can try the authentication flow.
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 8]
```diff
@@ -206,7 +206,7 @@ It looks like this:
 
 Add the following URL query parameters to the URL:
 
-- Your OpenID Connect application's `client_id` and `redirect_uri`
+- Your OpenID Connect app's `client_id` and `redirect_uri`
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:209]:
```suggestion
- Your OIDC app's `client_id` and `redirect_uri`
```

File: packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md [hunk 9]
```diff
@@ -223,7 +223,7 @@ If you paste this into your browser, you’re redirected to the sign-in page for
 
 `https://{yourOktaDomain}/login/login.htm?fromURI=%2Foauth2%2Fv1%2Fauthorize%2Fredirect%3Fokta_key%aKeyValueWillBeHere`
 
-Enter the credentials for a user who is mapped to your OpenID Connect application, and you’re directed to the `redirect_uri` that you specified. An ID token and any state that you defined are also included:
+Enter the credentials for a user who is mapped to your OpenID Connect app, and you’re directed to the `redirect_uri` that you specified. An ID token and any state that you defined are also included:
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md:226]:
```suggestion
Enter the credentials for a user who is mapped to your OIDC app, and you’re directed to the `redirect_uri` that you specified. An ID token and any state that you defined are also included:
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/dcb9bcefed5a74e33fa33feb6144caa18c656125/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9f96a6b3-0bc7-4968-bd01-5ccedd966649) | 82  | 78  | 95  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.df5f017e-a037-4807-ada6-8437b4994ecd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**